### PR TITLE
@/Roboshield Payload configurations. #760

### LINKED ISSUE #760: @/Roboshield Payload configurations.
SEO optimizations, images on blocks, and  partners list on site settings are missing. This needs to be added to improve Roboshield usability.

**Acceptance Criteria**
- [ ] All blocks should have images of the sections.
- [ ] Pages Collections should show - Title, full title and slug. ID is not required.
- [ ] Enable SEO in initial Set Up.
- [ ] Rename Statistics to something else.
- [x] #765
- [x] #766

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

This PR 
- [x] Adds preview images to blocks.
- [x] Updates pages collection default columns.
- [x] Renames CustomEmbedd block to CustomEmbed.
- [x] Adds titles to partners list

Fixes #760

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

Preview doesn't work for me.

**Comment by koechkevin** [comment]
> 👍🏽
> 
> Preview doesn't work for me.


> 👍🏽
> 
> Preview doesn't work for me.

Not sure which error you're getting @kilemensi

**Comment by kilemensi** [comment]
Not seeing any actual errors @koechkevin :

Screenshot 2024-07-11 at 17 11 42

And on terminal:
```
> roboshield@0.1.0 dev /Users/clemence/Work/github.com/CodeFor
Africa/ui/apps/roboshield
> NODE_OPTIONS='--inspect' ts-node --project tsconfig.server.j
son server.ts

Debugger listening on ws://127.0.0.1:9229/badcb8f1-ee0d-41ad-a
b69-65edbb688001
For help, see: https://nodejs.org/en/docs/inspector
[17:09:34] INFO (payload): Connected to MongoDB server success
fully!
[17:09:34] INFO (payload): Starting Payload...
[17:09:34] INFO (payload): Payload Admin URL: /admin
webpack built 3db6a9f60b906fd6be8d in 6710ms
webpack compiled successfully
[17:09:42] INFO (payload): NextJS started
[17:09:42] INFO (payload): Next.js App URL: undefined
<w> [webpack.cache.PackFileCacheStrategy] Caching failed for p
ack: Error: No serializer registered for ConcatSource
<w> while serializing webpack/lib/util/registerExternalSeriali
zer.webpack-sources/ConcatSource -> Array { 2 items } -> Conca
tSource
```

**Comment by kilemensi** [comment]
Fixed the URL issue @koechkevin but preview is still failing
```
> roboshield@0.1.0 dev /Users/clemence/Work/github.com/Code
ForAfrica/ui/apps/roboshield
> NODE_OPTIONS='--inspect' ts-node --project tsconfig.serve
r.json server.ts

Debugger listening on ws://127.0.0.1:9229/74d11430-6f5a-40c
f-87b1-31c3de59bf8e
For help, see: https://nodejs.org/en/docs/inspector
[10:19:24] INFO (payload): Connected to MongoDB server succ
essfully!
[10:19:24] INFO (payload): Starting Payload...
[10:19:24] INFO (payload): Payload Admin URL: /admin
[10:19:28] INFO (payload): NextJS started
[10:19:28] INFO (payload): Next.js App URL: http://localhos
t:3000
webpack built cef378508f67b61fcd98 in 6352ms
webpack compiled successfully
```

**Comment by koechkevin** [comment]
> Fixed the URL issue @koechkevin but preview is still failing
> 
> ```
> > roboshield@0.1.0 dev /Users/clemence/Work/github.com/Code
> ForAfrica/ui/apps/roboshield
> > NODE_OPTIONS='--inspect' ts-node --project tsconfig.serve
> r.json server.ts
> 
> Debugger listening on ws://127.0.0.1:9229/74d11430-6f5a-40c
> f-87b1-31c3de59bf8e
> For help, see: https://nodejs.org/en/docs/inspector
> [10:19:24] INFO (payload): Connected to MongoDB server succ
> essfully!
> [10:19:24] INFO (payload): Starting Payload...
> [10:19:24] INFO (payload): Payload Admin URL: /admin
> [10:19:28] INFO (payload): NextJS started
> [10:19:28] INFO (payload): Next.js App URL: http://localhos
> t:3000
> webpack built cef378508f67b61fcd98 in 6352ms
> webpack compiled successfully
> ```

`PAYLOAD_PUBLIC_APP_URL` @kilemensi

**Comment by kilemensi** [comment]
`PAYLOAD_PUBLIC_APP_URL` should default to `NEXT_PUBLIC_APP_URL` if not provided @koechkevin (or that's how I at least remember it from the [`Dockerfile`](https://github.com/CodeForAfrica/ui/blob/e1797d7d4301146fc2d05e3549cc88481e65f00d/Dockerfile#L385-L386))... but let me test that too.

**Comment by kilemensi** [comment]
That seems to do the trick @koechkevin. Please update the env file to have a [default](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#referencing-other-variables) `PAYLOAD_PUBLIC_APP_URL`. Something like 👇🏽 can work:
```sh
MIGRATIONS_DIR=./migrations

NEXT_PUBLIC_APP_URL=http://localhost:3000
NEXT_PUBLIC_IMAGE_DOMAINS="*.codeforafrica.org"
NEXT_PUBLIC_IMAGE_UNOPTIMIZED="true"

PAYLOAD_PUBLIC_APP_URL=$NEXT_PUBLIC_APP_URL
PAYLOAD_PUBLIC_DEFAULT_LOCALE=en
PAYLOAD_PUBLIC_LOCALES="en"

SENTRY_ENVIRONMENT=local
SENTRY_ORG=code-for-africa
SENTRY_PROJECT=roboshield

```

Someone can set `ENV` or use `.env.local` to override if they need to.

File: apps/roboshield/.env [hunk 1]
```diff
@@ -9,3 +9,5 @@ SENTRY_PROJECT=roboshield
 
 NEXT_PUBLIC_IMAGE_DOMAINS="*.codeforafrica.org"
 NEXT_PUBLIC_IMAGE_UNOPTIMIZED="true"
+
+NEXT_PUBLIC_APP_URL=http://localhost:3000
```
Comment (kilemensi) [inline apps/roboshield/.env:13]:
```suggestion
NEXT_PUBLIC_APP_URL=http://localhost:3000
PAYLOAD_PUBLIC_APP_URL=$NEXT_PUBLIC_APP_URL
```

i.e. PAYLOAD should use NEXT URL if not set.

**Review by kilemensi — APPROVED** [review]

**Pull request merged**